### PR TITLE
Update UploadQuery.java

### PR DESCRIPTION
**Pull request by SawsenHarhouri**

### CODE REVIEW

**Comment by TimBloembergen** [comment]
@SawsenHarhouri Can you put in a subscription for you changes? Thanks

**Comment by rcuisnier** [comment]
NB: This is linked to https://github.com/Bynder/bynder-java-sdk/pull/90

**Comment by rcuisnier** [comment]
NB: This is linked with https://github.com/Bynder/bynder-java-sdk/pull/90

File: src/main/java/com/bynder/sdk/query/upload/UploadQuery.java [hunk 1]
```diff
@@ -57,8 +64,31 @@ public String getFilepath() {
...
     }
 
+    public UploadQuery setFilepath(final String filePath) {
+        this.filepath = filePath;
+        return this;
+    }
+
     public String getFilename() {
-        return Paths.get(filepath).getFileName().toString();
+            return Paths.get(filepath).getFileName().toString();
```
Comment (Arpit-Sharma-USC) [inline src/main/java/com/bynder/sdk/query/upload/UploadQuery.java:73]:
```suggestion
        return Paths.get(filepath).getFileName().toString();
```

**Comment by Arpit-Sharma-USC** [comment]
Left a suggestion for you @SawsenHarhouri.
Also, please:
- add the change in the #90 within this PR. Since they both add 1 functionality they should go in a single PR
- add Unit tests for your changes

**Comment by Arpit-Sharma-USC** [comment]
@SawsenHarhouri @rcuisnier This was released with version 2.2.16 via #96

**Pull request closed without merging**